Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6-stretch
+
+WORKDIR /opt/avere-europarl
```

Comment:
This directory should exist before you set it as a `WORKDIR`: `RUN mkdir /opt/...`